modules/cluster-bare-metal-operator.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// *  operators/operator-reference.adoc
+// *  post_installation_configuration/cluster-capabilities.adoc
+
+// operators/operator-reference.adoc
+ifeval::["{context}" == "cluster-operators-ref"]
+:operator-ref:
+endif::[]
+
+// post_installation_configuration/cluster-capabilities.adoc
+ifeval::["{context}" == "cluster-capabilities"]
+:cluster-caps:
+endif::[]
+
+:_content-type: REFERENCE
+[id="cluster-bare-metal-operator_{context}"]
+ifdef::operator-ref[= Bare Metal Operator]
+ifdef::cluster-caps[= Bare-metal capability]
+
+ifdef::operator-ref[]
+
+[NOTE]
+====
+The Bare Metal Operator is an optional cluster capability that can be disabled by cluster administrators during installation. For more information about optional cluster capabilities, see "Cluster capabilities" in _Post-installation configuration_.
+====
+
+endif::operator-ref[]
+
+[discrete]
+== Purpose
+
+ifdef::cluster-caps[]
+
+The Bare Metal Operator provides the features for the `baremetal` capability.
+
+endif::cluster-caps[]
+
+The Bare Metal Operator (BMO) enables management of bare-metal hosts to be used as nodes in the cluster. When deploying OpenShift Container Platform on bare-metal hosts, you might need to make changes to the host either before or after provisioning.
+
+ifdef::cluster-caps[]
+In addition to managing bare-metal nodes, the BMO handles interactions with Ironic during installer-provisioned installations (IPI). Disabling the bare-metal capability can result in unexpected problems with IPI deployments.
+
+It is recommended that cluster adminstrators only disable the bare-metal capability during user-provisioned installations (UPI) that do not have any bare-metal nodes in the cluster. 
+
+[IMPORTANT]
+====
+If the bare-metal capability is disabled, users cannot interact with bare-metal clusters.
+====
```

Comment:
@elfosardo and @pamoedom Are these changes to the content accurate?

Should I change "unexpected problems" so something more specific?

Reply:
I have updated the content based on other feedback. Would you PTAL and let me know if the changes are correct?

Reply:
@elfosardo Would you please take a look at the changes and let me know if they are accurate?

Reply:
Hi @michaelryanpeter, thanks for the modification, lgtm with one exception:

Instead of saying `...user-provisioned installations (UPI) that do not have any bare-metal nodes in the cluster`, I would prefer something like `...user-provisioned installations (UPI) that do not have any BareMetalHost resource in the cluster`. @elfosardo, do you agree?

Please note that UPI BM also have baremetal nodes, the difference is that those nodes are not managed by the BMO but manually by the customer/admin.

Best Regards.

Reply:
@pamoedom @elfosardo : Thank you for the feedback. I have addressed your comments. PTAL.